src/core/mesh/qgsmesheditor.cpp

```diff
@@ -22,6 +22,7 @@
 #include "qgsgeometryengine.h"
 #include "qgsmeshadvancedediting.h"
 #include "qgsgeometryutils.h"
+#include "qgspolygon.h"
```

Comment:
Needed? Just checking.

Reply:
Yep, the header cleanup elsewhere in this pr changes things here.